src/helpers/getProjectConfig.js

```diff
@@ -0,0 +1,46 @@
+import path from 'path';
+import { readFile } from 'fs';
+import { promisify } from 'es6-promisify';
+import globToFiles from './globToFiles';
+
+// This function is used to handle the case where react-component-variation configs
+// are spread out across multiple files.
+// We expect each config file to be a json file that contains configuration under the field "react-component-variations"
+export default async function getProjectConfig(rootConfig) {
+  const { projectConfigPath } = rootConfig;
+  if (projectConfigPath) {
+    // Here we look through all the project config files that match our glob
+    // and then read each file and create a config object
+    return Promise.all(globToFiles(projectConfigPath).map(async (file) => {
+      const projectConfigFileContents = await promisify(readFile)(file);
+      const config = await JSON.parse(String(projectConfigFileContents))['react-component-variations'];
```

Comment:
JSON.parse does not return a promise, `await`ing it achieves nothing except making the function slower.

Separately, it's been pretty intentional not to hardcode this field name, since it can be overridden on the command line using yargs' `--config` option.